### ISSUE TITLE
[Todo] Improved range

### ISSUE THREAD

**Issue by zaydek** [zaydek/duomo#71]
```
  1
  2   4   6   8  10
 12  14  16  18  20 (add 13, 15, 17, 19)
 24  28  32  36  40
 48  56  64  72  80
 96 112 128 144 160
```

**Comment by zaydek** [comment]
This is more predictable and covers up to `160` instead of `128`.

**Comment by zaydek** [comment]
Implemented for 0.5.1 or 0.6.0.
